Source/AtomicEditor/Application/AEEditorCommon.cpp

```diff
@@ -255,4 +258,53 @@ bool AEEditorCommon::ReadPreferences()
     return true;
 }
 
+void AEEditorCommon::ValidateWindow()
+{
+    Graphics* graphics = GetSubsystem<Graphics>();
+    IntVector2 windowPosition = graphics->GetWindowPosition();
+    int monitors = graphics->GetMonitorsNumber();
+    IntVector2 maxResolution;
+
+    for (int i = 0; i < monitors; i++)
+    {
+        IntVector2 monitorResolution = graphics->GetMonitorResolution(i);
+        maxResolution += monitorResolution;
+    }
+
+    if (windowPosition.x_ >= maxResolution.x_ || windowPosition.y_ >= maxResolution.y_ || windowPosition.x_ < 0 || windowPosition.y_ < 0)
+    {
+        bool editor = this->GetTypeName() == "AEEditorApp";
```

Comment:
Better to use context_->GetEditorContext()

https://github.com/AtomicGameEngine/AtomicGameEngine/blob/master/Source/Atomic/Core/Context.h#L120